JsonFile.js

```diff
@@ -205,6 +263,8 @@ Object.assign(JsonFile, {
   deleteKeyAsync,
   deleteKeysAsync,
   rewriteAsync,
-});
+};
+
+Object.assign(JsonFile, fns);
```

Comment:
Everything should use locking I think -- we're using file locks so that the locking applies across JsonFile instances and even processes, right?